### ISSUE TITLE
Feature Request: Adjust Border Expansion Factor

### ISSUE THREAD

**Issue by faomy** [Return-To-The-Roots/s25client#1072]
Hi, 

would it be possible to consider adding an Addon to set a factor for border pushing to the different military buildings?  

I played some games on large / huge custom (512 x 512) maps and found the radius of expansions a little too low.  Plastering the map in Fortresses was a decent workaround, of course, but wondered, if

**Comment by Flow86** [comment]
Hi, 
currently, the default MILITARY_RADIUS is hard coded in MilitaryConsts.h but you could modify it at 

https://github.com/Return-To-The-Roots/s25client/blob/3c6e59b1556d1d154d16ed45a7ef4711f64b71bf/libs/s25main/buildings/nobMilitary.cpp#L304

@Flamefire 
at https://github.com/Return-To-The-Roots/s25client/blob/3c6e59b1556d1d154d16ed45a7ef4711f64b71bf/libs/s25main/buildings/nobMilitary.cpp#L335 I think `GetMilitaryRadius` should be used there instead of the constant?

**Comment by Flamefire** [comment]
> I think GetMilitaryRadius should be used there instead of the constant?

True. The function was introduced later, so this is an oversight in refactoring.

> I played some games on large / huge custom (512 x 512) maps

Note that such large maps might lead to problems in later game stages due to to many entities to handle which might cause lacks

**Comment by faomy** [comment]
Hi, thanks already for the lightning quick replies.

I'm using the Windows s25rttr_0.8.2 client and will try to find the referenced passage. 
And i never have had any experience with coding in particular, so 

> Note that such large maps might lead to problems in later game stages due to to many entities to handle which might cause lacks <

Thanks again for the advice. I took it into account the last time i posted on the rttr forums and reduced my size from 1024 x 1024  when i asked about saving issues :)  I will keep an eye out for issues and eventually adjust the size again.

**Comment by Flamefire** [comment]
It is suggested to use the nighlies. We wanted to release the current nightly as the next stable but didn't do that yet. However current nightlies are more stable than the stable ;)

**Comment by Spikeone** [comment]
Just saying: The border distance should (maybe) also affect the attack distance and (maybe) the minimum distance between military buildings.
